bootstrap.js

```diff
@@ -14,9 +14,9 @@ function startup(data, reason) {
 function shutdown(data, reason) {
   console.log('bootstrap shutdown called');
   // no teardown is needed for a normal shutdown
-  if (reason == APP_SHUTDOWN) { return; }
+  if (reason === APP_SHUTDOWN) { return; }
 
-  if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
+  if (reason === ADDON_DISABLE || reason === ADDON_UNINSTALL) {
```

Comment:
This is probably a case where it would be nice to define the constants as ignored globals at the top of the file, I think.